app/search_result/search_result.component.ts

```diff
@@ -122,12 +122,13 @@ export class SearchResultProcurementComponent implements OnInit {
   constructor() {}
 
   ngOnInit() {
-    this.details = 'לורם איפסום ' || this.item.source.title;
+    let item = this.item, source = item.source;
+    this.details = 'לורם איפסום ' || source.title;
 
     this.isTitleTextMatched = this.verifyTitleMatch();
```

Comment:
I think this function needs to be fixed too (to check both supplier_name and entity_name).